### ISSUE TITLE
New version broke the twitch

### ISSUE THREAD

**Issue by xeophyte** [davidofwatkins/rescroller#20]
New version 2024.7.7 broke the twitch left channel panel

![vivaldi - 2024 07 17_13 38 58](https://github.com/user-attachments/assets/0b7b4456-143a-4446-9483-74e2adf94174)

**Comment by xeophyte** [comment]
Any fix? Adding twitch.tv to Ignore list doesn't work for long.

**Comment by davidofwatkins** [comment]
Hi @xeophyte, I'm not seeing the same issue on Twitch. Did disabling or uninstalling Rescroller fix the issue? Adding `twitch.tv` to the ignore list in the extension settings prevents any of the extension code from being injected into the page, so I'm not sure why that wouldn't prevent the issue at least 🤔

It seems that Twitch has its own custom Javascript scrollbar that overrides Rescroller, so the extension isn't much good there anyway. I suppose it's possible that their scrollbar is conflicting with Rescroller somehow, but again, ignoring Twitch in Rescroller settings should address that.

**Comment by xeophyte** [comment]
Yes, disabling Rescroller fix the issue (after refreshing twitch page). 
Adding twitch.tv to the ignore list (and save settings) helps, but only until restarting the browser or disabling/enabling extension. After that it's broken again and I have to go to settings and save it again, every time.
Btw, the settings aren't saved after you type something to the list, you have to click to some other setting to save after that. 
Btw2, I've got also youtube.com added to the ignore list and it works even after restart the browser/extension, twitch.tv isn't.

I'm using Vivaldi browser v 6.9.3442.4.

**Comment by xeophyte** [comment]
Will this be fixed?
